controllers/access_control.go

```diff
@@ -576,6 +576,16 @@ func (roleCreate aerospikeRoleCreateUpdate) updateRole(
 		}
 	}
 
+	if role.ReadQuota != roleCreate.readQuota || role.WriteQuota != roleCreate.writeQuota {
```

Comment:
Add a test for this change.

Reply:
Setting, Re-setting

Reply:
test is already there https://github.com/aerospike/aerospike-kubernetes-operator/blob/9d3edf738a8ca65031fd617fdd5a50b4fd036b81/test/access_control_test.go#L1896